README.md

```diff
@@ -1,8 +1,48 @@
-# React + Vite
+# NC News
 
-This template provides a minimal setup to get React working in Vite with HMR and some ESLint rules.
+Welcome to my NC News Project!
 
-Currently, two official plugins are available:
+NC News is a frontend dynamic single-page React application designed to provide a seamless and engaging experience for browsing and interacting with articles and article comments.
 
-- [@vitejs/plugin-react](https://github.com/vitejs/vite-plugin-react/blob/main/packages/plugin-react/README.md) uses [Babel](https://babeljs.io/) for Fast Refresh
-- [@vitejs/plugin-react-swc](https://github.com/vitejs/vite-plugin-react-swc) uses [SWC](https://swc.rs/) for Fast Refresh
+## Key Features
+
+- **Browse Articles**: View all articles across various topics
+
+- **Sort and Order**: Easily sort and order articles by date, title, author and other criteria in ascending or descending order
+
+- **Topic Selection**: Choose specific topics to filter articles
+
+- **Article Voting**: Upvote or downvote Articles
+
+- **Comments Interaction**: Read and engage with comments on articles. Logged in users can post new comments.
+
+### Technologies
+
+Built using React.js, MUI Materials, Tailwind CSS, Daisy UI
+
+## Live Site 
+
+https://nc-news-melissalin.netlify.app/
+
+## Installation
```

Comment:
May want a lead-in here e.g. "if you want to run my repository locally follow these steps..." 👍